python/ola/PidStore.py

```diff
@@ -157,14 +161,50 @@ def ValidateAddressing(self, args, command_class):
         return False
     return True
 
-  def __cmp__(self, other):
-    return cmp(self._value, other._value)
+  def _GroupCmp(self, a, b):
+    def setToDescription(x):
+      def descOrNone(y):
+        return y.GetDescription() if y is not None else "None"
```

Comment:
I think this will break with the most recent changes. I don't think it's currently used anywhere but could be useful in the future, so up to you if you want to drop it or fix it?

Reply:
I fixed it, though as you said it doesn't seem to be referenced anywhere.  Put in a test case that I have no idea if it is what the intended use is, but at least confirmed it broke before the fix and works now.

Reply:
Thanks. It's only come in during this PR, I think in one of the previous iterations to fully compare two PIDs. But given it's written it seemed a shame to drop it.